[2025-10-07T10:16:41.240+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-07T10:16:41.350+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:01:57.519057+00:00 map_index=0 [queued]>
[2025-10-07T10:16:41.394+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:01:57.519057+00:00 map_index=0 [queued]>
[2025-10-07T10:16:41.394+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 3
[2025-10-07T10:16:41.458+0000] {taskinstance.py:2890} INFO - Executing <Mapped(_PythonDecoratedOperator): process_batch> on 2025-10-07 10:01:57.519057+00:00
[2025-10-07T10:16:41.517+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=287) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-07T10:16:41.527+0000] {standard_task_runner.py:72} INFO - Started process 461 to run task
[2025-10-07T10:16:41.649+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'vnstock_weekly_etl_incremental', 'process_batch', 'manual__2025-10-07T10:01:57.519057+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpge159ni5', '--map-index', '0']
[2025-10-07T10:16:41.662+0000] {standard_task_runner.py:105} INFO - Job 12: Subtask process_batch
[2025-10-07T10:16:41.761+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-07T10:16:42.234+0000] {task_command.py:467} INFO - Running <TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:01:57.519057+00:00 map_index=0 [running]> on host e3278e0f0bbc
[2025-10-07T10:16:50.306+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='vnstock_weekly_etl_incremental' AIRFLOW_CTX_TASK_ID='process_batch' AIRFLOW_CTX_EXECUTION_DATE='2025-10-07T10:01:57.519057+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-07T10:01:57.519057+00:00'
[2025-10-07T10:16:50.323+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-10-07T10:16:50.345+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý YTC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.346+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý YEG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.347+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý YBM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.348+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý YBC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.362+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý XPH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.374+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý XMP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.375+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý XMD: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.381+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý XMC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.382+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý XLV: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.387+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý XHC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.410+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý XDH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.411+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý XDC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.413+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý X77: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.415+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý X26: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.437+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý X20: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.439+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý WTC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.441+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý WSS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.443+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý WSB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.453+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý WCS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.456+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VXT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.461+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VXP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.470+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VXB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.471+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VWS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.482+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VW3: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.495+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VVS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.505+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VVN: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.514+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VUA: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.541+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTZ: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.542+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTX: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.543+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTV: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.544+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.551+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTR: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.554+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTQ: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.555+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.556+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTO: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.559+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.561+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTL: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.575+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTK: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.578+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTJ: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.579+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTI: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.599+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.599+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.600+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTE: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.601+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTD: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.601+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.607+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.613+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VTA: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.614+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VST: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.618+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VSP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.631+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VSN: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.646+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-07T10:16:50.680+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-10-07T10:16:50.686+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=vnstock_weekly_etl_incremental, task_id=process_batch, run_id=manual__2025-10-07T10:01:57.519057+00:00, map_index=0, execution_date=20251007T100157, start_date=20251007T101641, end_date=20251007T101650
[2025-10-07T10:16:50.911+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-07T10:16:51.352+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-07T10:16:51.373+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
